content/6_Advanced/Extend_Euclid.problems.json

```diff
@@ -24,19 +24,8 @@
       "isStarred": false,
       "tags": [],
       "solutionMetadata": {
-        "kind": "none"
-      }
-    },
-    {
-      "uniqueId": "kattis-generalchineseremainder",
-      "name": "Chinese Remainder (non-relatively prime moduli)",
-      "url": "https://open.kattis.com/problems/generalchineseremainder",
-      "source": "Kattis",
-      "difficulty": "N/A",
-      "isStarred": false,
-      "tags": [],
-      "solutionMetadata": {
-        "kind": "none"
+        "kind": "in-module",
+        "moduleId": "modular"
```

Comment:
fix module id

Reply:
done

Reply:
this isn't fixed yet...

Reply:
Ah sorry. I just added moduleId withouth modifying it :))

Reply:
Now it's ok